hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -409,10 +414,10 @@ public static boolean requireBlockToken(
   public static boolean requireContainerToken(
       ContainerProtos.Type cmdType) {
     switch (cmdType) {
+    case ListBlock:
```

Comment:
Can we leave the unsupported operation such as listContainer/listBlock as-is to get the proper error (unsupported) response to the caller instead of token validation error?

Reply:
@xiaoyuyao Can you please explain why this is important and what's the benefit of keeping the current behavior?

Reply:
Two reasons: 
1. We are sending token validation error for those unsupported operation, which makes it like the call is supported and we just need a valid token. The actual reason that the operation is not supported will be hidden after this.
2. We don't need to secure APIs that we don't support.  Some of those unsupported operations are added in early days when SCM is not there. When the implementation is added, I would suggest we review the security requirements case by case then.

Reply:
Thanks for the explanation.  Latest commit removes token requirement for these.